### ISSUE TITLE
containers don't show up in GUI

### ISSUE THREAD

**Issue by mas-dse-juremigi** [docker/kitematic#1250]
if you start a container, they no longer show up in the GUI like in previous versions...

**Comment by FrenchBen** [comment]
Can you share the version of Kitematic/docker you are using?
What does the following show:
`docker $(docker-machine config default) ps -a`

**Comment by mas-dse-juremigi** [comment]
kitematic version: 0.9.3

CONTAINER ID        IMAGE                          COMMAND
CREATED             STATUS              PORTS                    NAMES
c3e940fa38de        jupyter/datascience-notebook   "tini -- start-notebo"
6 days ago          Up 20 hours         0.0.0.0:8888->8888/tcp   cocky_pike

**Comment by FrenchBen** [comment]
To QA this issue I did the following from the command line:
`docker $(docker-machine config default) run kitematic/hello-world-nginx`
Immediately saw the container show up in kitematic.
Then ran the same command but setup as a daemon:
`docker $(docker-machine config default) run -itd kitematic/hello-world-nginx`
and also saw it in Kitematic.


Based on your output, I ran the following:

`docker $(docker-machine config default) run jupyter/datascience-notebook`
While the container was downloading I didn't see anything in kitematic, which is expected.
Once the container was actually up and running it showed up as expected within the container list.
